### ISSUE TITLE
Can't install from zip source

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Tangtry/gameobjects#2]
```
What steps will reproduce the problem?
1. Download 0.0.1 zip source file
2. Unpack
3. cd to source 
4. "python setup.py install"

This should install the source, but instead I get:

root@shiny:/tmp/gameobjects-0.0.1# python setup.py install
Traceback (most recent call last):
  File "setup.py", line 13, in <module>
    from gameobjects import __version__
ImportError: No module named gameobjects


The "package_dir = {'gameobjects': ''}" line in the setup file is causing 
the mis-packaging. Removing it (and moving all the unpacked source files 
to a "gameobjects" subdir) creates a correct sdist file and enables 
installation.

```

Original issue reported on code.google.com by `r1chardj0n3s` on 12 Jun 2007 at 9:27

**Comment by GoogleCodeExporter** [comment]
```
Ah, it worked here because I already had a gameobjects in my python path. I 
fixed it
by retrieving the version correctly, i.e.

from __init__ import __version__

Now that installing from zip source works, are there any benefits in moving the 
code
in to a gameobjects subdir? If not, I'm inclined to leave it as is.
```

Original comment by `willmcgugan` on 12 Jun 2007 at 9:50
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
It'd be a little cleaner for distribution.


Wow, I didn't know that distutils had a hack to prevent installation of the 
setup.py...
```

Original comment by `r1chardj0n3s` on 12 Jun 2007 at 10:34

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.0.2
```

Original comment by `willmcgugan` on 15 Jun 2007 at 7:41
- Changed state: **Verified**
